oemoflex/tools/colors.yaml

```diff
@@ -0,0 +1,13 @@
+PV: '#bcbd22'
```

Comment:
The order of the stackplot could be defined here. If a technology is listed earlier in here, it should appear closer to the x-axis in the stackplot.